sylvia/src/types.rs

```diff
@@ -38,13 +39,44 @@ impl<'a, C: cosmwasm_std::CustomQuery, Contract> From<&'a BoundQuerier<'a, C, Co
     }
 }
 
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[derive(Serialize, Deserialize, Derivative)]
+#[derivative(Clone, Debug, PartialEq)]
 pub struct Remote<'a, Contract> {
     addr: std::borrow::Cow<'a, cosmwasm_std::Addr>,
     #[serde(skip)]
+    #[derivative(Debug = "ignore")]
     _phantom: std::marker::PhantomData<Contract>,
 }
 
+// Custom `JsonSchema` implementation to remove bounds for `Contract`.
+const _: () = {
```

Comment:
Why is this in some block? rather than just a top level `impl`?

(Help me improve my Rust skills)

Reply:
This is how `JsonSchema` derive macro generates it. Seems not required